client/src/modules/vouchers/vouchers.service.js

```diff
@@ -188,5 +190,18 @@ function VoucherService(Api, $http, TransactionTypeStore, Modal,
     }).result;
   }
 
+  /**
+   * @method remove
+   *
+   * @description
+   * This function removes a voucher from the database via the transaction
+   * delete route.
+   */
+  function remove(uuid) {
```

Comment:
This code is seriously duplicated, no way to create a service to manage transaction as entity?

Reply:
@jniles 
This comment is not addressed.